python/kwiver/vital/types/bounding_box.cxx

```diff
@@ -38,9 +38,12 @@
 
 namespace py = pybind11;
 
-
+namespace kwiver {
```

Comment:
Adding namespace here seems a little redundant since only a typedef is being declared in the namespace